ex5.3_perfect_splite_meal.py

```diff
@@ -0,0 +1,32 @@
+def perfect_split():
+    """
+    generator that check if the number is perfect split
+    :return:
+    """
+    current_number = 0
+    while True:
+        if is_perfect(current_number):
+            yield current_number
+        current_number = current_number + 1
+
+
+def is_perfect(n):
+    """ If sum of divisors is equal to
```

Comment:
these sentences are too short - dont use that much line breaks, look at pep 8 max line length